pages/linux/zypper.md

```diff
@@ -14,10 +14,18 @@
 
 `zypper remove {{package}}`
 
-- Upgrade installed packages to newest available versions:
+- Upgrade installed packages to newest available versions on Leap:
 
 `zypper update`
 
+- Upgrade installed packages to newest available versions on Tumbleweed:
+  
+`zypper dist-upgrade`
+
+  Note: DO NOT update Tumbleweed through the YaST GUI or with `zypper update`
```

Comment:
We do not allow lines like this in the page format. If you think it is a _must have_, then you can add it along with the description in brackets. But I believe that will make it too long for a tldr page.